hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/keys/PutKeyHandler.java

```diff
@@ -83,8 +87,20 @@ protected void execute(OzoneClient client, OzoneAddress address)
       }
     }
 
-    ReplicationConfig replicationConfig =
-        ReplicationConfig.parse(replicationType, replication, getConf());
+    String clientConfiguredDefaultType = getConf().get(OZONE_REPLICATION_TYPE);
+    if (replicationType == null && clientConfiguredDefaultType != null) {
```

Comment:
Isn't this two condition remained here accidentally? Maybe I miss something, but if I am correct, we do the same in this two ifs as inside the method OzoneClientUtils#validateAndGetClientReplicationConfig(), which we call from here.

Reply:
Good catch. I already moved these checks inside OzoneClientUtils.validateAndGetClientReplicationConfig. 
There are just redundant here and removed them now.